Core/Sources/AblyExtensions/ARTRealtimeChannels+Extensions.swift

```diff
@@ -0,0 +1,14 @@
+import Ably
+
+private let channelNamePrefix = "tracking"
+
+extension ARTRealtimeChannels {
+    func getChannelFor(trackingId: String, options: ARTRealtimeChannelOptions? = nil) -> ARTRealtimeChannel {
```

Comment:
Is this extension accessible outside of our SDKs? I guess it shouldn't be visible for our SDK users but I don't know enough of iOS and Swift to tell if this is how it works now :wink:

Reply:
Thanks for mentioning this, currently its following the same style as another extension:
```swift
extension ARTRealtimeChannelState {
    func toConnectionState() -> ConnectionState {
        switch self {
        case .attached:
            return .online
        case .failed:
            return .failed
        default:
            return .offline
        }
    }
}
```
Because no access modifier is specified, it is by default **internal access** only.  If it is meant to public, you will should mark it as open or public. So this won't be accessible to other users, I confirmed it with: 

![Screenshot 2021-05-24 at 13 46 01](https://user-images.githubusercontent.com/24711048/119349738-632de300-bc96-11eb-8137-5b0dfc18ccd4.png)

Reply:
Awesome, thanks for checking :+1: